packages/engine/Source/Scene/VoxelCylinderShape.js

```diff
@@ -163,84 +143,45 @@ VoxelCylinderShape.prototype.update = function (
   Check.typeOf.object("maxBounds", maxBounds);
   //>>includeEnd('debug');
 
-  const defaultMinRadius = VoxelCylinderShape.DefaultMinBounds.x;
-  const defaultMaxRadius = VoxelCylinderShape.DefaultMaxBounds.x;
-  const defaultMinAngle = VoxelCylinderShape.DefaultMinBounds.y;
-  const defaultMaxAngle = VoxelCylinderShape.DefaultMaxBounds.y;
-  const defaultMinHeight = VoxelCylinderShape.DefaultMinBounds.z;
-  const defaultMaxHeight = VoxelCylinderShape.DefaultMaxBounds.z;
-  const defaultAngleRange = defaultMaxAngle - defaultMinAngle;
+  const { DefaultMinBounds, DefaultMaxBounds } = VoxelCylinderShape;
+  const defaultAngleRange = DefaultMaxBounds.y - DefaultMinBounds.y;
   const defaultAngleRangeHalf = 0.5 * defaultAngleRange;
 
   const epsilonZeroScale = CesiumMath.EPSILON10;
   const epsilonAngleDiscontinuity = CesiumMath.EPSILON3; // 0.001 radians = 0.05729578 degrees
   const epsilonAngle = CesiumMath.EPSILON10;
 
-  // Clamp the radii to the valid range
-  const shapeMinRadius = CesiumMath.clamp(
-    minBounds.x,
-    defaultMinRadius,
-    defaultMaxRadius,
-  );
-  const shapeMaxRadius = CesiumMath.clamp(
-    maxBounds.x,
-    defaultMinRadius,
-    defaultMaxRadius,
-  );
-  const clipMinRadius = CesiumMath.clamp(
-    clipMinBounds.x,
-    defaultMinRadius,
-    defaultMaxRadius,
-  );
-  const clipMaxRadius = CesiumMath.clamp(
-    clipMaxBounds.x,
-    defaultMinRadius,
-    defaultMaxRadius,
-  );
-  const renderMinRadius = Math.max(shapeMinRadius, clipMinRadius);
-  const renderMaxRadius = Math.min(shapeMaxRadius, clipMaxRadius);
-
-  // Clamp the angles to the valid range
-  const shapeMinAngle = CesiumMath.negativePiToPi(minBounds.y);
-  const shapeMaxAngle = CesiumMath.negativePiToPi(maxBounds.y);
-  const clipMinAngle = CesiumMath.negativePiToPi(clipMinBounds.y);
-  const clipMaxAngle = CesiumMath.negativePiToPi(clipMaxBounds.y);
-  const renderMinAngle = Math.max(shapeMinAngle, clipMinAngle);
-  const renderMaxAngle = Math.min(shapeMaxAngle, clipMaxAngle);
-
-  // Clamp the heights to the valid range
-  const shapeMinHeight = CesiumMath.clamp(
-    minBounds.z,
-    defaultMinHeight,
-    defaultMaxHeight,
-  );
-  const shapeMaxHeight = CesiumMath.clamp(
-    maxBounds.z,
-    defaultMinHeight,
-    defaultMaxHeight,
-  );
-  const clipMinHeight = CesiumMath.clamp(
-    clipMinBounds.z,
-    defaultMinHeight,
-    defaultMaxHeight,
+  minBounds = Cartesian3.clone(minBounds, this._minBounds);
+  maxBounds = Cartesian3.clone(maxBounds, this._maxBounds);
+
+  // Clamp the bounds to the valid range
+  minBounds.x = Math.max(0.0, minBounds.x);
+  maxBounds.x = Math.max(0.0, maxBounds.x);
+  minBounds.y = CesiumMath.negativePiToPi(minBounds.y);
+  maxBounds.y = CesiumMath.negativePiToPi(maxBounds.y);
+
+  clipMinBounds.y = CesiumMath.negativePiToPi(clipMinBounds.y);
+  clipMaxBounds.y = CesiumMath.negativePiToPi(clipMaxBounds.y);
+
+  // TODO: what if a min angle is > a max angle? (crossing the 180th meridian)
```

Comment:
Discussed offline, but since this is an edge case, we can document this as a known issue rather than fix this here. Please open a new issue, if you haven't done so already. 🙂

Reply:
False alarm: I tested this, and clipping across the antimeridian actually works fine!
I removed the TODO comments.